src/Quadrature.jl

```diff
@@ -399,23 +399,27 @@ ZygoteRules.@adjoint function __solvebp(prob,alg,sensealg,lb,ub,p,args...;kwargs
     function quadrature_adjoint(Δ)
         y = typeof(Δ) <: Array{<:Number,0} ? Δ[1] : Δ
         if isinplace(prob)
-            if lb isa Number && prob.batch == 0
-                dx = Float64[lb]
-            elseif lb isa Number
-                dx = zeros(length(lb),prob.batch)
-            elseif prob.batch == 0
-                dx = zeros(length(lb))
-            else
-                dx = zeros(length(lb),prob.batch)
-            end
-            _f = (x) -> (prob.f(dx,x,p); dx)
+            # if lb isa Number && prob.batch == 0
+            #     dx = Float64[lb]
+            # elseif lb isa Number
+            #     dx = zeros(length(lb),prob.batch)
+            # elseif prob.batch == 0
+            #     dx = zeros(length(lb))
+            # else
+            #     # dx = zeros(length(lb),prob.batch)
+            #     dx = zeros(prob.nout, prob.batch)
+            # end
+            # _f = (x) -> (prob.f(dx,x,p); dx)
+
+            # @show size(dx)
 
             if sensealg.vjp isa ZygoteVJP
-                dfdp = function (dx,x,p)
+                # dfdp = function (dx,x,p)
+                dfdp = function (x,p)
```

Comment:
Changed to oop b/c unsure how to copy result to dx w/o mutating. If using `Buffer` we need to allocate for the result anyway